### PR TITLE
updated tests to remove duplicate code

### PR DESCRIPTION
**Pull request by noagb**
Resolves #NUMBER

**Overall change:**
_Refactored Previous Social Media embeds test with simpler code._

**Code changes:**

- _simplified the test so that there is no repetition of the same code._

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] I have added the `cross-team` label to this PR if it requires visibility across World Service teams
- [x] I have assigned this PR to the Simorgh project
- [x] (BBC contributors only) This PR follows the [repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)

**Testing:**

Executes on both Test and Live Env

### CODE REVIEW

File: cypress/integration/pages/articles/tests.js [hunk 1]
```diff
@@ -190,128 +190,57 @@ export const testsThatFollowSmokeTestConfig = ({
...
-                    cy.get(`[href^="#end-of-tiktok-content"]`).should('exist');
+        const socialMediaProviders = [
+          'YouTube',
+          'Instagram',
+          'TikTok',
+          'Twitter',
+          'Facebook',
+        ];
+        socialMediaProviders.forEach(socialMediaProviderName => {
+          it.only(`${socialMediaProviderName} embed is redered when it exists on page`, () => {
```
Comment (LilyL0u) [inline cypress/integration/pages/articles/tests.js]: Will need to remove the .only

Comment (LilyL0u) [inline cypress/integration/pages/articles/tests.js]: Just a lil typo on the word 'rendered'

File: cypress/integration/pages/articles/tests.js [hunk 2]
```diff
@@ -190,128 +190,57 @@ export const testsThatFollowSmokeTestConfig = ({
...
-          cy.request(`${Cypress.env('currentPath')}.json`).then(({ body }) => {
-            const facebookSocialEmbedsData = getAllSocialBlocksByProviderName(
-              'Facebook',
-              body,
+                } else {
+                  cy.log(
+                    `There is no ${socialMediaProviderName} embed in this page!`,
+                  );
+                  cy.get(
+                    `[href^="#end-of-${lowercaseSocialMediaProviderName}-content"]`,
```
Comment (LilyL0u) [inline cypress/integration/pages/articles/tests.js]: This needs to be inside the if clause, as it is running even if there isn't an embed on the page and then the test fails.

**Review by LilyL0u — DISMISSED** [review]

**Review by LilyL0u — APPROVED** [review]
I like this refactor - efficient!

**Review by amoore108 — APPROVED** [review]

**Pull request merged**